app/logic/events.py

```diff
@@ -507,6 +508,79 @@ def getEventRsvpCount(eventId):
     """
     return len(EventRsvp.select().where(EventRsvp.event_id == eventId))
 
+def getCountdownToEvent(event, *, currentDatetime=None):
+    """
+    Given an event, this function returns a string that conveys the amount of time left
+    until the start of the event.
+
+    Note about dates:
+    The way dates are communicated is unintuitive. There are two major rules that govern how we discuss dates.
+    - If an event happens tomorrow but less than 24 hours away from us we still say that it happens 
+    tomorrow with no mention of the hour. 
+    - If an event happens tomorrow but more than 24 hours away from us, we'll count the number of days 
+    and hours in actual time.
+
+    E.x. if the current time of day is greater than the event start's time of day, we give a number of days 
```

Comment:
Since I have already left other comments I will add this nitpicky thing, I believe it is E.g. not E.x. if you are meaning to abbreviate "for example".

Reply:
Finn, I see that leaving comments that aren't absolutely 100% correct isn't in your skillset. Good catch